Makefile

```diff
@@ -135,5 +144,6 @@ help:
 	$(PRINT) 'CARGO_OPTS          --  Changes `cargo` default behavior (e.g. --verbose).'
 	$(PRINT) 'NOVENDOR            --  Avoid to self detect and consume the vendored headers'
 	$(PRINT) '                        shipped with libbpf-sys.'
+	$(PRINT) 'COV                 --  enable code coverage for builds.'
```

Comment:
Here, given the reasons above (rustup toolchain), we should make it clear that the feature is supported only for toolchains installed via rustup and that is applicable only for `test`